08-Placing-Barriers-in-the-Simulation-Box/05-Wall-potential-HPMC.ipynb

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
<p>"...the previous section show<strong>ed</strong> how to run ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>